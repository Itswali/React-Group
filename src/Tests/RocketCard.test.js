/* eslint-disable import/no-extraneous-dependencies */
import TestRenderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketCard from '../components/RocketCard';

describe('Render the ROcket Reservation Section ', () => {
  it('Renders all Section with rocket reservaton or cancelation', () => {
    const missions = TestRenderer.create(
      <Provider store={store}>
        <RocketCard />
      </Provider>,
    ).toJSON();
    expect(missions).toMatchSnapshot();
  });
});
