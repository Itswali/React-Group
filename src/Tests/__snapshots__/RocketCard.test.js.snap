// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the ROcket Reservation Section  Renders all Section with rocket reservaton or cancelation 1`] = `
<div
  className="p-2 hstack gap-2 minBreakpoint-xs"
>
  <div>
    <img
      alt="rocket"
      className=""
      width={250}
    />
  </div>
  <div
    className="vstack minBreakpoint-xs"
  >
    <h2 />
    <p />
    <div>
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Render the missions list in profile page Renders all joined missions correctly into the profile page 1`] = `
<div
  className="p-2 hstack gap-2 minBreakpoint-xs"
>
  <div>
    <img
      alt="rocket"
      className=""
      width={250}
    />
  </div>
  <div
    className="vstack minBreakpoint-xs"
  >
    <h2 />
    <p />
    <div>
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;
